build-dev/README.md

```diff
@@ -0,0 +1,9 @@
+# Start dev env as easy as possible
+
```

Comment:
one general comment: should we put this under /tools?

Reply:
move to tools/devenv